common/common-rest/src/test/java/org/apache/servicecomb/common/rest/TestAbstractRestInvocation.java

```diff
@@ -104,13 +110,23 @@ protected OperationLocator locateOperation(ServicePathManager servicePathManager
     }
 
     @Override
-    protected void createInvocation(Object[] args) {
+    protected void createInvocation() {
       this.invocation = TestAbstractRestInvocation.this.invocation;
     }
   }
 
   AbstractRestInvocation restInvocation = new AbstractRestInvocationForTest();
 
+  @BeforeClass
+  public static void classSetup() {
+    EventManager.eventBus = new EventBus();
+  }
+
+  @AfterClass
+  public static void classTeardown() {
+    EventManager.eventBus = new EventBus();
```

Comment:
Do we need to setup new EventBus again here?

Reply:
no
out eventBus extend from this one, but faster.